core/src/main/java/it/pagopa/selfcare/external_api/core/ContractServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package it.pagopa.selfcare.external_api.core;
+
+
+import it.pagopa.selfcare.commons.base.logging.LogUtils;
+import it.pagopa.selfcare.external_api.api.FileStorageConnector;
+import it.pagopa.selfcare.external_api.api.MsCoreConnector;
+import it.pagopa.selfcare.external_api.model.documents.ResourceResponse;
+import it.pagopa.selfcare.external_api.model.token.Token;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class ContractServiceImpl implements ContractService {
+    private final FileStorageConnector fileStorageConnector;
+    private final MsCoreConnector msCoreConnector;
+
+    @Autowired
```

Comment:
@Autowired annotation could be omitted here, Spring automatically inject objects in constructor